### PR TITLE
Separate File* models

### PR DESCRIPTION
**Pull request by beezz**

### CODE REVIEW

File: src/sentry/routers.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+
+def db_for_model(model):
+    db_table = model._meta.db_table
+    if db_table in {
```
Comment (markstory) [inline src/sentry/routers.py:6]: If you define this set at the module level you'll only create it once. Right now the set is built on each method call.

**Review by markstory — COMMENTED** [review]
Will file tables be in the same or separate database for development environments and CI? Having CI and ideally local development aligned with how prod operates will help ensure that we don't accidentally introduced a joined load somewhere in the future.

File: src/sentry/api/endpoints/debug_files.py [hunk 2]
```diff
@@ -405,12 +405,10 @@ def expose_release(release, count):
...
 
         def serialize_results(results):
-            file_counts = (
-                Release.objects.filter(id__in=[r["id"] for r in results])
-                .annotate(count=Count("releasefile"))
-                .values("count", "id")
-            )
-            file_count_map = {r["id"]: r["count"] for r in file_counts}
+            file_count_map = {
+                release.id: ReleaseFile.objects.filter(release=release).count()
```
[the remark this answers is not in the document]
Comment (beezz) [reply src/sentry/api/endpoints/debug_files.py:409]: Good point, I can optimize that and fire just one query for all the releases.

File: src/sentry/routers.py (hunk 1 above)
Comment (beezz) [reply src/sentry/routers.py:6]: Initially, that was meant as a throw-away code but if we decide to have multiple DBs also in sentry dev/ci I will change it as suggested.

**Pull request closed without merging**